kotlin/internal/jvm/jvm.bzl

```diff
@@ -344,6 +344,10 @@ Setup a simple kotlin_test.
             default = Label("@bazel_tools//tools/jdk:TestRunner_deploy.jar"),
             allow_files = True,
         ),
+        "use_testrunner": attr.bool(
```

Comment:
What exactly is this flag solving for and when would you use it externally vs. setting it to `False`?

Reply:
It matches [java_test.use_testrunner](https://bazel.build/reference/be/java#java_test.use_testrunner). It allows customizing the test execution environment. In this case, we replace the bazel test runner with JUnit5.